TreeDB/cmd/treedb_rag_benchmark/application_compare.go

```
@@ -0,0 +1,1056 @@
+	hnswM, hnswMOK := artifact.Server.Config["hnsw_m"].(float64)
+	indexingThreshold, indexingThresholdOK := artifact.Server.Config["indexing_threshold"].(float64)
+	if artifact.Server.Version != manifest.Config.QdrantServerVersion || artifact.Server.LocalMode ||
+		artifact.Server.Deployment != "standalone" ||
+		artifact.Server.BinarySHA256 != manifest.Config.QdrantBinarySHA256 ||
+		artifact.Server.Config["dense"] != manifest.Config.DenseVectorName ||
+		artifact.Server.Config["sparse"] != manifest.Config.SparseVectorName ||
+		artifact.Server.Config["full_scan_threshold_unit"] != "KiB" ||
+		!exactOK || exact || !fullScanOK || fullScanThresholdKB != 10 ||
+		!hnswMOK || hnswM != 16 || !indexingThresholdOK || indexingThreshold != 1 ||
+		artifact.Server.IndexProof.IndexedVectorsCount < 2*len(manifest.Chunks) {
+		return fmt.Errorf("Qdrant artifact lacks pinned standalone server/index configuration")
```

golangci-lint (2.13.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)